ims/company/lost_or_stolen.md

```diff
@@ -0,0 +1,24 @@
+# What To Do When Your Device is Lost or Stolen
+
+The first thing you should do if your device is lost or stolen is report the incident to the CSO or IMS Manager as soon as possible.
+
+For Team members using the standard build of Made Tech device, we are able to use MDM software to remotely wipe data from your device. 
```

Comment:
We should explain acronyms or link to definitions, methinks.

Reply:
MDM has an explanation. 

IMS manager and CSO have been explained elsewhere in the IMS File